src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+| their_channel_reserve_sats         | boolean | Optional, defaults to `10000` or 1% of channel value. The minimum balance that the other node has to maintain on their side, at all times. This ensures that if our counterparty broadcasts a revoked state, we can punish them by claiming at least this value on chain. |
+
+
+<Note>
+For GUIs and wallet apps, it is recommended to set `announced_channel` to `false` (the default value), as the node is not expected to be reliably online. 
+</Note>
+
+### LightningChannelOptions
+
+
+| Parameter                              | Type    | Description                                                                                   |
+|----------------------------------------|---------|-----------------------------------------------------------------------------------------------|
+| proportional_fee_in_millionths_sats    | integer | Optional, defaults to `0`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| base_fee_msat                          | integer | Optional, defaults to `1000`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| cltv_expiry_delta                      | integer | Optional, defaults to `72`. Blocks until [CheckLockTimeVerify (CLTV)](https://academy.bit2me.com/en/que-es-cltv-bitcoin/) expiry.                                |
+| max_dust_htlc_exposure_msat            | integer | Optional, defaults to `5000000`. Limit our total exposure to in-flight [HTLCs](https://academy.binance.com/en/glossary/hashed-timelock-contract) which are burned to fees as they are too small to claim on-chain.  |
+| force_close_avoidance_max_fee_sats     | integer | Optional, defaults to `1000`. The additional fee we're willing to pay to avoid waiting for the counterparty's locktime to reclaim funds.                            |
+
+
+### LightningClosedChannelsFilter
+
+| Parameter                   | Type    | Description                                                                                                     |
+|-----------------------------|---------|-----------------------------------------------------------------------------------------------------------------|
+| channel_id                  | string  | Optional. Unique string identifying a channel by its ID.                                                        |
+| counterparty_node_id        | string  | Optional. A hexidecimal string identifying a counterparty node. |
+| funding_tx                  | string  | Optional. A transaction ID which added funds.                   |
+| from_funding_value          | integer | Optional. The minimum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| to_funding_value            | integer | Optional. The maximum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
```

Comment:
Details for closed channels differ from open channels as stated in a previous comment, this name is consistent with the `funding_value` in closed channel details. If it's not available in the closed channel details response, it means that the channel was rejected by us if it was inbound or was rejected by the counterparty if it was outbound before it was even opened, meaning that there was no funding transaction.

Reply:
DevNote added to track this